content/100-getting-started/02-setup-prisma/100-start-from-scratch/110-relational-databases/100-connect-your-database.mdx

```diff
@@ -51,7 +51,7 @@ As an example, for a PostgreSQL database hosted on Heroku, the [connection URL](
 DATABASE_URL="postgresql://opnmyfngbknppm:XXX@ec2-46-137-91-216.eu-west-1.compute.amazonaws.com:5432/d50rgmkqi2ipus?schema=hello-prisma"
 ```
 
-When running PostgreSQL locally on Mac OS, your user and password as well as the database name _typically_ correspond to the current _user_ of your OS, e.g. assuming the user is called `janedoe`:
+When running PostgreSQL locally on macOS, your user and password as well as the database name _typically_ correspond to the current _user_ of your OS, e.g. assuming the user is called `janedoe`:
```

Comment:
I like this fix - the format for "macOS" should be in our style guide. I'll add it.

Reply:
I see you opened https://github.com/prisma/docs/pull/3948
Nice!